frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelSupport.tsx

```diff
@@ -90,30 +103,27 @@ const SupportFormBlock = ({ onCancel }: { onCancel: () => void }): JSX.Element =
                         <Link to={urls.organizationBilling([ProductKey.PLATFORM_AND_SUPPORT])}>Explore options</Link>
                     </div>
                 </div>
-                {billing?.products
-                    ?.find((product) => product.type == ProductKey.PLATFORM_AND_SUPPORT)
-                    ?.plans?.map((plan) => (
+                {supportPlans?.map((plan) => {
+                    // If they have an addon plan, only show the addon plan
```

Comment:
I'm entirely sure I understand this conditional for currentPlan. 

If your current plan is `teams-20240208` and the billing changes aren't merged yet, `hasAddonPlan == false` and `plan.plan_key.includes('addon') == false` so `currentPlan == false`. Does that mean none of the rows in the table are marked (your plan)?

Reply:
If they have the teams addon that means they also have either the paid or enterprise platform plan. We don't want to show your plan twice. So this is meat to avoid that.

If we ship these changes and none of the billing ones it'll still work because addons will be an empty array
![Screenshot 2024-04-29 at 2 37 34 PM](https://github.com/PostHog/posthog/assets/6392501/96cf7366-6d6b-452d-96f7-e7a7224da593)
addon then